AppCenter/AppCenterTests/MSCSExtensionsTests.m

```diff
@@ -490,6 +490,13 @@ - (void)testAppExtIsEqual {
 
   // Then
   XCTAssertNotEqualObjects(anotherAppExt, self.appExt);
+
+  // If
+  anotherAppExt.locale = self.appExtDummyValues[kMSAppLocale];
+  anotherAppExt.userId = @"c:charlie";
```

Comment:
You need to add userId to `MSModelTestsUtililty` class. Take a look at `appExtensionDummies` method in the class.